skit.Infrastructure/Integrations/Emails/Services/EmailSenderService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using MailKit.Net.Smtp;
+using MimeKit;
+using skit.Infrastructure.Integrations.Emails.Configuration;
+
+namespace skit.Infrastructure.Integrations.Emails.Services;
+
+public sealed class EmailSenderService : IEmailSenderService
+{
+    private readonly SmtpConfig _smtpConfig;
+
+    public EmailSenderService(SmtpConfig smtpConfig)
+    {
+        _smtpConfig = smtpConfig;
+    }
+    
+    public async Task SendEmailAsync(string email, string subject, string textBody)
+    {
+        var emailMessage = CreateEmailMessage(email, subject, textBody, _smtpConfig);
+        using var client = new SmtpClient();
+        {
+            await client.ConnectAsync(_smtpConfig.SmtpUrl, _smtpConfig.SmtpPort, false);
+            await client.AuthenticateAsync(_smtpConfig.SmtpLogin, _smtpConfig.SmtpPassword);
+            var status = await client.SendAsync(emailMessage);
+            await client.DisconnectAsync(true);
+        }
+    }
```

Comment:
Testowałeś już czy taka prosta autoryzacja działa? Wiem, że ostatnio były z tym problemy bo wprowadzono jakieś ograniczenia i już tak łatwo nie dało się autoryzować z pocztą

Reply:
Identyczną miałem u siebie w projekcie i wszystko działało. Ale jeszcze wyjdzie przy dorabianiu tych maili

Reply:
Kwestia stestowania tego, bo sam próbowalem w swoim wczesniejszym projekcie robic praktycznie identycznie i nie podolalem przez jakies dziwne blokady jakie mialy nalozone skrzynki pocztowe. Na gmailu nie potrafilem zrobic, a na interii działało ale tylko pojedyczne wyslanie maila, po 1 mailu skrzynka sie blokowala i musialem recznie zaznaczac ze skrzynka jest bezpieczna czy cos takiego. Takze pasuje to sprawdzic czy cokolwiek mozna wyslac